README.md

```diff
@@ -48,13 +51,22 @@ https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/oauth/OIDCClai
 used to define tags in the AWS session.  The tags are names `synapse-`<claim name>, where <claim name> is the name of the claim given in the config file.
 
 The `SESSION_NAME_CLAIMS` config is also a comma separated list of claims, but used to define the session name, as a colon delimited list of claim values. For example: setting `SESSION_NAME_CLAIMS=userid,email` will display
-`ServiceCatalogEndusers/1234567:joe.smith@gmail.com` in AWS. 
+`ServiceCatalogEndusers/1234567:joe.smith@gmail.com` in AWS.
+
+Note:  The list of claims requested from Synapse is the union of the two lists, `SESSION_TAG_CLAIMS` and `SESSION_NAME_CLAIMS`, plus the `userid` claim, which this application uses itself.
```

Comment:
> plus the `userid` claim, which this application uses itself.

Does this mean that `userid` is required to be on the   SESSION_NAME_CLAIMS list?  if so could we reword it to be more explicit?

Reply:
> Does this mean that userid is required to be on the SESSION_NAME_CLAIMS list?

No. The app' takes the union of SESSION_NAME_CLAIMS, SESSION_TAG_CLAIMS and `userid` to get the list of things to ask Synapse for.  The configuration of SESSION_NAME_CLAIMS and SESSION_TAG_CLAIMS need not include userid.